release/cibuild.py

```diff
@@ -265,18 +297,24 @@ def should_upload_pypi(self) -> bool:
         ])
 
     @property
-    def tag(self):
-        return self.travis_tag or self.appveyor_repo_tag_name
+    def tag(self) -> str:
+        if self.travis_tag:
```

Comment:
Do we still need this - and the one below for appveyor?

Reply:
Likely no, but I would keep this until we are certain that GitHub Actions does everything we want.